packages/builder/src/pages/[application]/_reset.svelte

```diff
@@ -27,6 +27,17 @@
       throw new Error(pkg)
     }
   }
+
+  const topItemNavigate = path => () => {
```

Comment:
What was happening here?

Reply:
ah yes - iv added some comments

Reply:
cool, makes sense -  I wonder if routify has some kind of first-class API for this? Couldn't see anything in the docs